packages/sdk/tests/actions/doesAutoProtocolFeeSharesBuyback.ts

```diff
@@ -0,0 +1,11 @@
+import { type Address, getAbiItem } from "viem";
+import { publicClient } from "../globals.js";
+import { IComptroller } from "../../../abis/src/abis/IComptroller.js";
```

Comment:
Wrong import path. Must've been autocompleted incorrectly. Not sure how to fix that... 

But this should be `@enzymefinance/abis/IComptroller`